agroal-springframework/src/main/java/io/agroal/springframework/AgroalDataSourceConfiguration.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2020 Red Hat, Inc. and individual contributors as indicated by the @author tags.
+// You may not use this file except in compliance with the Apache License, Version 2.0.
+
+package io.agroal.springframework;
```

Comment:
As this is spring boot specific, maybe the package name should reflect that?

Reply:
Changed module name to `agroal-spring-boot-starter` and package name to `io.agroal.springframework.boot`